src/main/webapp/pages/projects/settings/pages/details.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html
+  lang="en"
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+  data-layout-decorate="~{projects/settings/_base}"
+>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Title</title>
+    <link rel="stylesheet" href="/dist/css/project-details.bundle.css">
+  </head>
+  <body>
+    <div id="details-root" layout:fragment="settings-content" >
+      <div>LOADING ...</div>
```

Comment:
Internationalize the text

Reply:
Updated in dea2b0460f709bf2d12e16db08482f38c7a98642